src/theme/Command.js

```diff
@@ -21,49 +21,100 @@ function insertDashDashBeforeOption(value, name) {
 
 export const CreateTauriApp = () => {
   return (
-    <Tabs className="">
+    <Tabs groupId="package-manager">
       <TabItem value="Bash">
-        <CodeBlock className={`language-shell`}>
+        <CodeBlock className="language-shell" language="shell">
           {`sh <(curl https://create.tauri.app/sh)`}
         </CodeBlock>
       </TabItem>
       <TabItem value="PowerShell">
-        <CodeBlock className={`language-powershell`}>
+        <CodeBlock className="language-shell" language="powershell">
           iwr -useb https://create.tauri.app/ps | iex
         </CodeBlock>
       </TabItem>
       <TabItem value="Cargo">
-        <CodeBlock className={`language-shell`}>
+        <CodeBlock className="language-shell" language="shell">
           {`cargo install create-tauri-app
 cargo create-tauri-app`}
         </CodeBlock>
       </TabItem>
       <TabItem value="npm">
-        <CodeBlock className={`language-shell`}>npm create tauri-app</CodeBlock>
+        <CodeBlock className="language-shell" language="shell">
+          npm create tauri-app
+        </CodeBlock>
       </TabItem>
       <TabItem value="Yarn">
-        <CodeBlock className={`language-shell`}>
+        <CodeBlock className="language-shell" language="shell">
           yarn create tauri-app
         </CodeBlock>
       </TabItem>
       <TabItem value="pnpm">
-        <CodeBlock className={`language-shell`}>
+        <CodeBlock className="language-shell" language="shell">
           pnpm create tauri-app
         </CodeBlock>
       </TabItem>
     </Tabs>
   )
 }
 
-export default ({ name, shell = 'shell' }) => {
+export const InstallTauriCli = () => {
+  return (
+    <Tabs groupId="package-manager">
+      <TabItem value="npm" default>
+        <CodeBlock className="language-shell" language="shell">
+          npm install @tauri-apps/cli
```

Comment:
This should be a dev dependency

```suggestion
          npm install --save-dev @tauri-apps/cli
```

same for other package managers